src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <ul class="calendar calendar--start-day-mon calendar--month-length-31">
+      <li class="calendar__day calendar__day--1"></li>
```

Comment:
You don't need to create separate classes for each day( `calendar__day--1`, `calendar__day--2`, etc. ), they can be accessed as children of the parent component.